dist/rei-dot-com/scss/cdr-tokens.scss

```diff
@@ -2284,6 +2284,144 @@ $cdr-membership-subtle-color-border-button-primary-rest: #13352c;
 $cdr-membership-subtle-color-border-button-primary-active: #13352c;
 $cdr-membership-subtle-color-border-button-primary-hover: #13352c;
 
+// @supports statements can be removed once Safari 15 is no longer supported
+
+// XS
+@mixin cdr-xs-cq-up($name: '') {
+  @container #{$name} (min-width: #{$cdr-breakpoint-xs}) {
+    @content;
+  }
+
+  @supports not (contain: inline-size) {
+    @media (min-width: #{$cdr-breakpoint-xs}) {
+      @content;
+    }
+  }
+}
+
+@mixin cdr-xs-cq-only($name: '') {
+  @container #{$name} (min-width: #{$cdr-breakpoint-xs}) and (max-width: calc(#{$cdr-breakpoint-sm} - 1px)) {
```

Comment:
as we move to fluid is using a fixed unit like px going to conflict with gap spacing that uses rem?

Reply:
Hmm, I don't think so, but maybe we can chat tomorrow about this, because I may not full understand